client/packages/system/src/Encounter/utils.ts

```diff
@@ -39,3 +44,16 @@ export const useEncounterFragmentWithStatus = (
     [nodes]
   );
 };
+
+export const useLogicalStatus = (
+  startDate: Date,
+  t: ReturnType<typeof useTranslation>
+): string | undefined => {
+  if (DateUtils.isToday(startDate)) {
```

Comment:
Have to check today, so status will display as `Past`

Reply:
It just checks the date and doesn't account for time... so as long as the time is within today, it won't show up as a past encounter... What do you consider as 'just missed'?

Reply:
For example, if the encounter was scheduled in the morning I would expect that it is missed as marked as missed in the afternoon so that the doctor, for example, could call the patient...

So I would compare the times rather than the dates.